packages/daemon/__tests__/integration/config.ts

```diff
@@ -0,0 +1,20 @@
+// Db information
+export const DB_USER = 'root';
+export const DB_PASS = 'hathor';
+export const DB_NAME = 'hathor';
+export const DB_PORT = 3380;
+export const DB_ENDPOINT = '127.0.0.1';
+
+// unvoided
+export const UNVOIDED_SCENARIO_PORT = 8081;
+export const UNVOIDED_SCENARIO_LAST_EVENT = 38;
+
+// reorg
+export const REORG_SCENARIO_PORT = 8082;
+export const REORG_SCENARIO_LAST_EVENT = 19;
+
+// single chain blocks and transactions port
+export const SINGLE_CHAIN_BLOCKS_AND_TRANSACTIONS_PORT = 8083;
+export const SINGLE_CHAIN_BLOCKS_AND_TRANSACTIONS_LAST_EVENT = 37;
```

Comment:
Yes, they're short by one because we don't store event ids on the database when we ignore an event and on all scenarios, the last event is an event we ignore

I will add a comment explaining this

Reply:
Done in 97ead7b1a9f691a24d6f79da2e9ded84449c9b7f

Reply:
Thanks!